binding.zig

```diff
@@ -19,16 +19,6 @@ const std = @import("std");
 const builtin = @import("builtin");
 const log = std.log.scoped(.OpenGL);
 
-pub const FunctionPointer: type = blk: {
-    const BaseFunc = fn (u32) callconv(.C) u32;
-    const SpecializedFnPtr = FnPtr(BaseFunc);
-    const fnptr_type = @typeInfo(SpecializedFnPtr);
-    var generic_type = fnptr_type;
-    std.debug.assert(generic_type.Pointer.size == .One);
-    generic_type.Pointer.child = anyopaque;
-    break :blk @Type(generic_type);
-};
-
```

Comment:
`FunctionPointer` is still required, as otherwise, we won't compile on any architecture that doesn't align functions at 1 (arm, aarch64, riscv, ...)

Reply:
Can it be `align(@alignOf(*const fn whatever_was)) anyopaque` instead, to show it more clearly?

Reply:
Should work as before now:
```zig
Compile Log Output:
@as(type, *align(4) const anyopaque)
@as(type, *align(4) const anyopaque)
```